app/src/main/java/com/example/our_space/LocationSelection.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2016 Jared Rummler
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.jaredrummler.materialspinner.example;
```

Comment:
This might have to do with the CI error

Reply:
Completely agree with Alex that this line is one of the problems with Travis CI